identity-verification/module_2/2.2 Onboarding IDV API Example.ipynb

```diff
@@ -0,0 +1,614 @@
+   "source": [
+    "### Setup Clients \n",
+    "-----\n",
+    "Here we are going to use both S3 and Rekognition apis  "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "s3_client  = boto3.client('s3')\n",
+    "rek_client = boto3.client('rekognition')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 0. Setup collections \n",
+    "-----\n",
+    "Here we will setup two collections \"registered-users\" and \"fraudulent-users\". The collections will be used to ensure that the user hasn't previously registered and that the user is not a known fraudster. Here we'll quickly create the two collections and index a handful of images. \n",
+    "\n",
+    "<div class=\"alert alert-info\"><strong> Avoiding Duplicates & Fraudsters </strong>\n",
+    "    \n",
+    "In practice, we use collections for many purposes. One is to avoid duplicate registrations; Without some form of biometric based onboarding, it is pretty easy for a user to create multiple accounts on a system simply by using different email addresses. Using a collection prevents this, because you can simply search the collection to see if a user has previously onobarded. Collections have proven to be effective at identifying and dealing with duplicate onboarding attempts as well as identifying fraudsters and bad actors. \n",
+    "\n",
+    "</div>\n",
+    "\n",
+    "\n",
+    "<div class=\"alert alert-info\"><strong> Create two Collections </strong>\n",
+    "\n",
+    "Here we are going to create two collections \"idv-registered-users\" and \"idv-fraudulent-users\" we'll load both collections with some images, and in step 6 and 7 of our process we'll check that the user hasn't previously onboarded. \n",
+    "\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "## Name your Collection \n",
+    "\n",
+    "# onboarded users collection\n",
+    "collection_name = 'idv-registered-users'\n",
+    "\n",
+    "try:\n",
+    "    rek_client.delete_collection(\n",
+    "        CollectionId=collection_name\n",
+    "    )\n",
+    "    rek_client.create_collection(\n",
+    "        CollectionId=collection_name\n",
+    "    )\n",
+    "except:\n",
+    "    response = rek_client.create_collection(\n",
+    "        CollectionId=collection_name\n",
+    "    )\n",
+    "\n",
+    "# fraudulent user collection \n",
+    "fraud_collection_name = 'idv-fraudulent-users'\n",
+    "\n",
+    "try:\n",
+    "    rek_client.delete_collection(\n",
+    "        CollectionId=fraud_collection_name\n",
+    "    )\n",
+    "    rek_client.create_collection(\n",
+    "        CollectionId=fraud_collection_name\n",
+    "    )\n",
+    "except:\n",
+    "    response = rek_client.create_collection(\n",
+    "        CollectionId=fraud_collection_name\n",
+    "    )"
```

Comment:
Similar to above, what if these names clash with existing resources in a customers account?

Reply:
updated.